autode/transition_states/templates.py

```diff
@@ -94,33 +310,82 @@ def save_object(self, basename='template', folder_path=None):
         # Iterate i until the templatei.obj file doesn't exist
         name, i = basename + '0', 0
         while True:
```

Comment:
minor thing, but I think this checks template0.txt twice?

Reply:
Not sure it does(?)